content/home/examples/a-component-using-external-plugins.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: A Component Using External Plugins
+title: Komponentas su įskiepiais.
 order: 3
 domid: markdown-example
 ---
 
-React allows you to interface with other libraries and frameworks. This example uses **remarkable**, an external Markdown library, to convert the `<textarea>`'s value in real time.
+_React_ leidžia turėti sąsają su kitomis bibliotekomis bei karkasais. Šis pavyzdys naudoja Markdown biblioteką **remarkable**, ir realiu laiku konvertuoją `<textarea>` esantį tekstą.
```

Comment:
`_React_ leidžia turėti sąsają su kitomis bibliotekomis bei karkasais. Šis pavyzdys naudoja Markdown biblioteką **remarkable**, ir realiu laiku konvertuoja `<textarea>` esantį tekstą.`